utils/ort/src/main/kotlin/JavaBootstrapper.kt

```diff
@@ -58,7 +59,11 @@ object JavaBootstrapper {
     fun isRunningOnJdk(version: String): Boolean {
         val requestedVersion = Semver.coerce(version)
         val runningVersion = Semver.coerce(Environment.JAVA_VERSION)
-        return requestedVersion == runningVersion
+        if (requestedVersion != runningVersion) return false
+
+        val javaHome = System.getProperty("java.home") ?: return false
+        val javac = File(javaHome).resolve("bin").resolve("javac")
+        return Os.resolveWindowsExecutable(javac) != null
```

Comment:
I was wondering why this is not wrapped with some `if (Os.isWindows)` check, but looking at the function it seems that on Linux it just checks if the file exists. Maybe the function should be renamed to just `resolveExecutable`, reading it here like this is unexpected.